lib/util.js

```diff
@@ -3,9 +3,12 @@
  */
 'use strict';
 
-const _ = require('lodash');
 const config = require('./config');
 const delay = require('delay');
+const lodashGet = require('lodash.get');
```

Comment:
so just to make sure this gets around the `lodash` deprecation notice on these modules somehow?

https://lodash.com/per-method-packages

> Per Method Packages
> Lodash methods are available in standalone per method packages like lodash.mapvalues, lodash.pickby, etc. These packages contain only the code the method depends on.
> 
> However, use of these packages is discouraged and they will be removed in v5.
> 
> Although they may seem more lightweight, they will usually increase the size of node_modules and webpack/rollup bundles in a project that transitively depends on multiple per method packages and/or the main lodash package. Whereas many methods in the main lodash package share code, the per method packages internally bundle copies of any code they depend on.
> 
> For example, throttle uses debounce internally. In a project using both methods from the main lodash package, throttle will import the same debounce module as any code that imports debounce directly, so only one copy of debounce will wind up in a webpack bundle.
> 
> On the other hand, if a project imports throttle from lodash.throttle, the extra copy of the debounce code internally bundled into lodash.throttle will wind up in the webpack bundle, in addition to debounce from the main lodash package or lodash.debounce.
> 
> But lodash isn't lightweight enough!
> Don't worry—if you import or require methods directly, e.g. const throttle = require('lodash/throttle'), only the subset of lodash code your package uses will be bundled in projects that use your package.
> 
> If importing this way seems cumbersome, you can use babel-plugin-lodash to transform named top-level imports like import { throttle, debounce } from 'lodash' into direct import statements.
> 
> Furthermore, modern tree-shaking bundlers like webpack and rollup can avoid bundling code you don't need even if you don't use direct imports or the babel plugin.
> 
> Migrating to the main lodash package
> A jscodeshift transform is available to convert per method package imports to main lodash package imports.

Reply:
@aljones15 this is good information.  However, they are addressing a different set of concerns here.  We're not doing this to reduce bundle size.  We're doing this mainly to not have bedrock getting tagged as unsafe the next time some random Lodash API has a CVE posted against it.  I think we can continue running these packages without lodash v5 from now until the end of time if we so choose.

Reply:
1. so this would mean if a CVE is found with `lodash.get` and `lodash.set` we would then need to upgrade 2 libraries?
2. Perhaps more concerning is that we are going to leave older code assuming it safe. I think being upgrade friendly is a better way of mitigating security risks than assuming all vulnerabilities have been found in older modules.
3. lodash is super popular and used in everything out there, this is a very real possibility that we will need newer methods in the future.

You have 2 approvals, but I'm not convinced on this one, but would prefer not to block if other people have ok'd this.